pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -283,12 +305,24 @@ func (a *ofFlowAction) Group(id GroupIDType) FlowBuilder {
 		Switch: a.builder.Flow.Table.Switch,
 		ID:     uint32(id),
 	}
-	a.builder.ofFlow.lastAction = group
+	a.builder.ApplyAction(group)
+	return a.builder
+}
+
+// Note adds notes to the OpenFlow entry. The notes are presented as hex digits in the OpenFlow entry, and it will be
```

Comment:
nit: s/Note adds notes to the OpenFlow entry./Note annotates the OpenFlow entry.

Reply:
Done.